core/src/main/java/org/infinispan/configuration/cache/PersistenceConfiguration.java

```diff
@@ -69,12 +69,11 @@ public int connectionInterval() {
    /**
     * Loops through all individual cache loader configs and checks if fetchPersistentState is set on
     * any of them
+    *
+    * @deprecated since 14.0. This will always return false
     */
+   @Deprecated
    public Boolean fetchPersistentState() {
```

Comment:
wondering why it returns a `Boolean` object...

Reply:
A wonder I hope I never need to find out :) But in all seriousness, guessing just a mistake.